src/parser.rs

```diff
@@ -76,9 +76,14 @@ fn parse_output<'a>(tokens: &Vec<Token>) -> Result<Box<Renderable + 'a>> {
             return Err(Error::Parser("parse_output: expected a colon".to_owned()));
         }
 
+        iter.next(); // skip colon
+
+        println!("{:?}", tokens);
```

Comment:
Didn't notice that debug print statement before, this is probably not supposed to be there, right? :)